test/PackageManagement.Test/BuildIntegratedRestoreUtilityTests.cs

```diff
@@ -408,6 +356,81 @@ public class BuildIntegratedRestoreUtilityTests
         }
 
         [Fact]
+        public async Task CacheHasChanges_ReturnsTrue_IfSupportProfilesDiffer()
```

Comment:
It's less meaningful though - the cache differs all the time - we just choose to return true if specific aspects of it are different. I'll leave this as is.